android/app-example/src/main/res/layout/rib_switcher.xml

```diff
@@ -8,9 +8,9 @@
     android:layout_height="match_parent"
     tools:context=".app.RootActivity">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
+    <!--<androidx.constraintlayout.widget.ConstraintLayout-->
```

Comment:
remove?

Reply:
I'll remove in separate ticket, as it relates to Blocker should be shown through Portal, and requires more work than this scope. Now it's quick fix to display correctly.